src/App.tsx

```diff
@@ -17,17 +24,23 @@ import { Select } from './ino-select';
 const NavItemLink = ({
   label,
   to,
+  activeOnlyWhenExact = true,
 }: {
   label: string;
   to: string;
   activeOnlyWhenExact?: boolean;
 }) => {
+  let match = useRouteMatch({
+    path: to,
+    exact: activeOnlyWhenExact,
+  });
   return (
-    <div>
-      <Link className={'router-link'} to={to}>
-        <InoNavItem inoText={label}></InoNavItem>
-      </Link>
-    </div>
+    <Link className={'router-link'} to={to}>
+      <InoNavItem inoActivated={match ? true : false} inoText={label}>
```

Comment:
Wouldn't `inoActivated={match}` be enough? Or something like `inoActivated={Boolean(match)}`? It feels kinda redundant the way it is

Reply:
I thought so as well but 👍 but `useRouteMatch` returns either an object or null :) so I think that's an easy way to handle it ;)

Reply:
Okay, but that sounds like `Boolean(match)` is exactly what we are looking for as it returns `false` if the value is `null`, empty, ... and `true` if it's an object 🤓 

it's really just a little detail but it would be interesting to me if this does not work here